doc/api-extensions.md

```diff
@@ -1406,7 +1406,7 @@ Add a new USBAddress (usb\_address) field to ResourcesGPUCard (GPU entries) in t
 
 ## clustering\_evacuation
 Adds `POST /1.0/cluster/members/<name>/state` endpoint for evacuating and restoring cluster members.
-It also adds the config keys `cluster.evacuate` and `volatile.evacuate.origin` for setting the evacuation method (`auto`, `stop` or `migrate`) and the origin of any migrated instance respectively.
+It also adds the config keys `cluster.evacuate` and `volatile.evacuate.origin` for setting the evacuation method (`auto`, `live-migrate`, `stop` or `migrate`) and the origin of any migrated instance respectively.
```

Comment:
We shouldn't alter past API extensions unless we're fixing a typo.

For this feature we should introduce a new API extension, let's call it `clustering_evacuation_live` which will need its own section in this file and in `shared/version/api.go`.